source/hic_hal/target_config.h

```diff
@@ -55,7 +55,8 @@ typedef struct target_cfg {
     uint8_t erase_reset;            /*!< Reset after performing an erase */
 } target_cfg_t;
 
-extern const target_cfg_t target_device;
+extern target_cfg_t target_device;
+void set_target_device(uint32_t device);
```

Comment:
Is it still OK to change the 'extern target_cfg_t target_device' to be non-const? Or will i have to hack it using pointers to get it set correctly in prerun_board_config?